packages/babel-plugin-remove-graphql-queries/src/index.js

```diff
@@ -76,12 +83,12 @@ function removeImport(tag) {
     else importPath.remove()
   }
   if (importPath.isObjectProperty()) {
-    if (parent.node.properties.length === 1)
-      importPath.findParent(p => p.isVariableDeclaration())?.remove()
```

Comment:
These were causing eslint to go wild. I _lightly_ refactored this and used a helper.